oscplot.c

```diff
@@ -2148,7 +2155,8 @@ static gboolean check_valid_setup_of_device(OscPlot *plot, const char *name)
 			gtk_widget_set_tooltip_text(priv->capture_button,
 				"Time Domain needs at least one channel");
 			return false;
-		} else if (dev && !dma_valid_selection(name, enabled_channels_mask | global_enabled_channels_mask(dev), nb_channels)) {
+		} else if (dev && !dma_valid_selection(name, enabled_channels_mask |
```

Comment:
It's no longer necessary to check 'dev' since you added a check already which will exist this function.